src/common_types.ts

```diff
@@ -119,6 +119,7 @@ export type CliArgs = {
   outfd?: number;
   headless?: boolean;
   screenshots?: boolean;
+  ssblocks?: boolean;
```

Comment:
@andrewvc If you have any better name for enabling the screenshot block capability.